 <launch>
   <node pkg="siamesenet" type="siameseNet.py" name="siameseNet" output="screen"/>
 </launch>
